bindings/aws/s3/metadata.yaml

```diff
@@ -0,0 +1,102 @@
+      description: "List blob"
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: bucket
+    required: true
+    description: |
+      The name of the S3 bucket to write to.
+    example: '"bucket"'
+    type: string
+  - name: region
+    required: true
+    description: |
+      The AWS region to the instance. See this page for valid regions:
+      https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html. 
+    example: '"us-east-1"'
+    type: string
+  - name: endpoint
+    required: false
+    description: |
+      AWS endpoint for the component to use, to connect to S3-compatible services or emulators.
+      Do not use this when running against production AWS.
+    example: '"http://localhost:4566"'
+    type: string
+  - name: sessionToken
+    required: false
+    sensitive: true
+    description: |
+      AWS session token to use. A session token is only required if you are using 
+      temporary security credentials.
+    example: '"TOKEN"'
+    type: string
+  - name: forcePathStyle
+    description: |
+      Currently Amazon S3 SDK supports virtual-hosted-style and path-style access.
+      When false (the default), uses virtual-hosted-style format, i.e.: `https://<your bucket>.<endpoint>/<key>`. 
+      When true, uses path-style format, i.e.: `https://<endpoint>/<your bucket>/<key>`.
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: decodeBase64
+    description: |
+      Configuration to decode base64 file content before saving to bucket storage.
+      (In case of saving a file with binary content). true is the only allowed
+      positive value. Other positive variations like "True", "1" are not acceptable. 
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: encodeBase64
+    description: |
+      Configuration to encode base64 file content before returning the content.
+      (In case of opening a file with binary content).
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: disableSSL
+    description: |
+      Allows to connect to non-`https://` endpoints. 
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: insecureSSL
+    description: |
+      When connecting to `https://` endpoints, accepts self-signed or invalid certificates.
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: filePath
+    required: false
+    description: |
+      To upload a file from a supplied path (relative or absolute), use the filePath
+      metadata key on a create request that contains empty data fields.
+    example: '"mydir/my-test-file.txt"'
+    type: string
+  - name: presignTTL
+    required: false
+    description: |
+      To presign an object with a specified time-to-live, use the presignTTL metadata
+      key on a create request, as a Go duration.
+    example: '"2h45m"'
+    type: duration
```

Comment:
According to the source code and the docs, `filePath` and `presignTTL` are keys used in the metadata of requests, not for the component. They should not be listed here.

https://docs.dapr.io/reference/components-reference/supported-bindings/s3/